src/h.js

```diff
@@ -28,4 +28,3 @@ export default function(tag, data) {
       }
     : tag(data, children)
 }
-
```

Comment:
It added a newline for me when I ran `yarn run format` after deleting the line on purpose.

But yes agreed, somehow this got missed :P